src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1507,6 +1554,17 @@ int sd_dhcp_server_set_default_lease_time(sd_dhcp_server *server, uint32_t t) {
         return 1;
 }
 
+int sd_dhcp_server_set_ipv6_only_preferred_time(sd_dhcp_server *server, uint32_t t) {
```

Comment:
Please clarify the unit of the input value is seconds: e.g. rename function `preferred_time()` -> `preferred_sec()` or rename the argument `t` -> `sec`.